Readme.MD

```diff
@@ -194,12 +197,17 @@ If one or more of your campaigns trigger when the `canDisplayCampaigns` property
 It is possible to get information about the feedback the user has left and you will also be notified if the user has left the form without submitting it, simply by implementing:
 
 ```swift
-func campaignDidClose(withFeedbackResult result: FeedbackResult, isRedirectToAppStoreEnabled: Bool) {
-    //...
-}
+func campaignDidClose(withFeedbackResult result: FeedbackResult, isRedirectToAppStoreEnabled: Bool) {}
 ```
 Unlike the Passive feedback method, the campaigns method only returns one [FeedbackResult](#feedbackresult) and is called only once.
 
+### Resetting all campaigns
+
+During development and testing, it might be useful to reset the status of your campaigns and allow them to be displayed again.    
+You can do so by calling `Usabilla.resetCampaignData(completion: nil)`.    
+You can include a closure that will be executed when SDK is done resetting the data.    
+This method will synchronize your data with the Usabilla server, so it will also fetch any new campaign as well as removing any deactivated one.
```

Comment:
...so it will fetch all the new campaigns and remove the deactivated ones as well.